ui/apps/platform/src/hooks/useAnalytics.ts

```diff
@@ -54,27 +54,29 @@ export const LEGACY_CLUSTER_DOWNLOAD_HELM_VALUES = 'Legacy Cluster Download Helm
  */
 type AnalyticsBoolean = 0 | 1;
 
-// search categories and type guards for tracking search filters on the Workload CVE pages
-export const searchCategoriesWithFilter = ['COMPONENT SOURCE', 'SEVERITY', 'FIXABLE'] as const;
+/**
+ * A curated list of filters that we would like to track both the filter category and the
+ * filter value. This list should exclude anything that could be considered sensitive or
+ * specific to a customer environment. This items in this list must also match the casing of
+ * the applied filter _exactly_, otherwise it will be tracked without the filter value.
+ */
+export const searchCategoriesWithFilter = [
+    'Component Source',
+    'SEVERITY',
+    'FIXABLE',
+    'CLUSTER CVE FIXABLE',
+    'CVSS',
+    'Node Top CVSS',
+] as const;
+
 export const isSearchCategoryWithFilter = tupleTypeGuard(searchCategoriesWithFilter);
 export type SearchCategoryWithFilter = UnionFrom<typeof searchCategoriesWithFilter>;
 
-export const searchCategoriesWithoutFilter = [
-    'CVE',
-    'IMAGE',
-    'COMPONENT',
-    'DEPLOYMENT',
-    'NAMESPACE',
-    'CLUSTER',
-] as const;
-export const isSearchCategoryWithoutFilter = tupleTypeGuard(searchCategoriesWithoutFilter);
-export type SearchCategoryWithoutFilter = UnionFrom<typeof searchCategoriesWithoutFilter>;
-
```

Comment:
The changes in the section serve the following purposes.

    Update the list of filter categories that are acceptable to also track the value. The only real addition here are the CVSS categories, which could be valuable to know what thresholds customers typically care about. The list also is changed to match the casing of both the filter values declared in the compound search filters utils and the ALL CAPS severity and fixability filters declared inline.

Ideally the casing of the severity and fixability filters would match, but changes to this in the past have resulted in buggy behavior that is unfortunately not caught at compile time.

I opted to still enforce that the event that includes the search value requires an exact match here, instead of just a case insensitive match. This keeps the telemetry strict and can help at compile time to ensure that bad values are not leaked. The downside here is that case-insensitive matching values may be tracked with only the category name, if we are not careful. I think a decent approach to avoid this is to handle testing telemetry as part of our e2e suite.

    Loosen the restrictions on the events without values to accept any string as a category. With advanced filters, the possible list is quite long now. Additionally, the plan moving forward is to decentralize the lists of search fields so it makes less sense to define this type in terms of strings that will be spread across the Containers directory. The downside here is that a developer error could allow bogus search categories to be track (but this would likely be a true bug in the search as well, which is worse and should be caught more quickly.)